packages/mesos-modules/extra/journal.json

```diff
@@ -0,0 +1,50 @@
+{
+  "libraries": [
+    {
+      "file": "libjournaldlogger.dll",
```

Comment:
https://github.com/dcos/dcos/blob/8c6ec915d7653a0fd288148c391454097f3f4bf2/gen/dcos-config.yaml#L378-L421

Reply:
I guess this comment is asking should these files be created by `dcos-config-windows.yaml`? This would provide more consistency between the two platforms.

Reply:
right